webapp/package.json

```diff
@@ -3,7 +3,8 @@
   "description": "The web application for the CentralDogma admin page",
   "license": "Apache-2.0",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "jest --watch",
+    "test:ci": "jest --ci",
```

Comment:
Could we add a Gradle task that triggers this task when `gradle test` is executed?
For example:
```gradle
task testWeb(type: NpmTask) {
    ...
}
tasks.test.dependsOn(tasks.testWeb)
```